.github/workflows/ci.yml

```diff
@@ -103,6 +106,9 @@ jobs:
       - name: Build Tempo
         run: make tempo
 
+      - name: generate-manifest
+        run: go run -v pkg/docsgen/generate_manifest.go
```

Comment:
Should this be a make target?

Reply:
personally I'm not a fan of putting this kind of commands in the makefile as it tends to grow a lot, but I'm fine with it if it's the team convention

Reply:
I rather use make as well. That's its main purpose

Reply:
Something to keep in mind is that every time we run make, some commands are run which add an overhead:
```
# Version number
VERSION=$(shell ./tools/version-tag.sh | cut -d, -f 1)

GIT_REVISION := $(shell git rev-parse --short HEAD)
GIT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
```

For now this is negligible, but as the Makefile grows we are more likely to introduce significant overhead. I've seen this in my previous job where we had a giant Makefile, that's why I'm averse to it and I would rather not use make as a generic task runner.